test/phoenix_live_view/integrations/live_view_test.exs

```diff
@@ -251,6 +251,13 @@ defmodule Phoenix.LiveView.LiveViewTest do
     end
   end
 
+  describe "private" do
```

Comment:
If this approach is good I can add a test for mount hooks to make sure private values are propagated correctly (I haven't tested hooks yet)